commerce-theme-minium/commerce-theme-minium/src/css/minium/components/mini_cart.scss

```diff
@@ -28,11 +28,15 @@
 
 	&:not(.is-open) {
 		box-shadow: 0 0 0 $color-shadow;
-		transform: translateX(100%);
+		transform: translateX(calc(100% + #{$liferay-sidebars-width}));
+	}
+
+	.rtl {
+		left: 0;
 	}
 
 	.rtl &:not(.is-open) {
-		transform: translateX(-100%);
+		transform: translateX(calc(-1 * calc(100% + #{$liferay-sidebars-width})));
```

Comment:
There is no need to nest "calc": parenthesis are enough. 
Btw writing `-100% - #{var}` I think it's just more readable (up to you :) )

Reply:
@FabioDiegoMastrorilli Multiplying by `-1` allows to read for negatives and it's a good practice to maintain both negative pure values and values in css/scss vars (which cannot be "negativized" straightforwardly in `calc`'s just by putting a `-` before them).

And that's the reason for the nesting of `calc`'s, there's no extra parenthesis.
Anyway I've just decided to go with a different way, as this improves readability *and* maintainability even more I think. Let me know.